src/test/resources/examples/differentErrorResponseBody/api.yaml

```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.0
```

Comment:
I think it would be cleaner if we just incorporated this additional test case into the existing example:
`okHttpClientMultiMediaType`